data/initrd/scripts/prepare_rescue

```diff
@@ -1,12 +1,37 @@
 #! /bin/sh
 
-# ensure some trees are completely writable
+# Ensure some directory trees are completely writable.
+#
+# ... but /etc/systemd will be treated specially (see below): move it aside
+# for now.
+#
```

Comment:
I know it's somewhat paranoid, but what if /etc_systemd already exists? Maybe add

    rm -rf /etc/systemd

to make sure it doesn't.